board/miyoo/scripts/genimage.sh

```diff
@@ -14,6 +14,7 @@ cp -r board/miyoo/main "${BINARIES_DIR}"
 mkdir -p "${BINARIES_DIR}/gmenu2x"
```

Comment:
why we make a dir here, can u recall?

Reply:
I think in the eariel version gmenu2x package produced only bin file, but I'm not sure. Now I think we can remove it